db/structure.sql

```diff
@@ -4657,7 +4657,7 @@ CREATE TABLE upvs.services_with_forms (
     meta_is_code character varying,
     name character varying,
     type character varying,
-    institution_uri character varying NOT NULL,
+    institution_uri character varying,
```

Comment:
Na taketo zmeny treba spravit migraciu, nie len upravit tu. Je vobec ale toto potrebne? Je tam taky zaznam? Co som narychlo pozrela, tak som nenasla taky. Ale upresni prosim.

Reply:
pri niektorom zazname v tom novom .csv to failovalo, ale skusim este pozriet a ked tak spravit migraciu

Reply:
@tomasdrga mas konkretny priklad kde to failovalo? Ja som nenasla (ani mi velmi nedava zmysel, aby pri tomto datasete bolo zverejnene nieco bez URI institucie.